pytest-selenium/tests/test_book.py

```diff
@@ -1,13 +1,20 @@
 # flake8: noqa
 import pytest
 from selenium.common.exceptions import NoSuchElementException, TimeoutException
+from selenium.webdriver.common.by import By
+
+import re
+from time import sleep
 
 from tests import markers
 from pages.content import Content
 from pages.osweb import WebBase
+from utils import utility
 from utils.utility import Utilities
```

Comment:
Instead of doing two separate imports, combine them:
`from utils.utility import Utilities, get_search_term`